dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/OpenApiActions.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.openapi20.fat;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.FeatureSet;
+import componenttest.rules.repeater.FeatureSet.FeatureSetBuilder;
+import componenttest.rules.repeater.JakartaEE9Action;
+import componenttest.rules.repeater.MicroProfileActions;
+import componenttest.rules.repeater.RepeatActions;
+import componenttest.rules.repeater.RepeatTests;
+
+public class OpenApiActions {
```

Comment:
Is this copy of OpenApiActions the same as the first one? I guess there is nowhere common for it to go? Does it matter if they use the same ID??

Reply:
Yes they're the same. At the moment we don't have anywhere common to put them, we would either need to make a new project just for this class or add it to the `fattest.simplicity` project.

While it's only this class, I'm ok having it duplicated.